### DISCUSSION TITLE
Welcome to the Docker Scout (Early Access) cli!

### DISCUSSION THREAD

**Discussion by jonnyrobots** [docker/scout-cli#5] [Announcements]
**A quick note to announce the arrival of the Docker Scout CLI** 🎉 

Please take a look at the reference documentation to learn more about commands like  `docker scout **compare**`, `docker scout **cves**`, and `docker scout **recommendations**`.

We're really looking forward to seeing what you create with `docker scout`, and be sure to let us know below what you'd like to be able to create as we evolve Docker Scout on the command-line!

**Comment by fai8146767** [comment]
****
